packages/matrix-client-server/src/login/getLogin.ts

```diff
@@ -0,0 +1,10 @@
+import { type expressAppHandler, send } from '@twake/utils'
+import type MatrixClientServer from '..'
+
+const getLogin = (clientServer: MatrixClientServer): expressAppHandler => {
+  return (req, res) => {
+    send(res, 200, clientServer.conf.login_flows)
```

Comment:
There is more than one authorized login flow. The more important for us is the "Openid-Connect". There are libraries to do this. I can explain the flow
@BichraiX @h1ppox99 @Mathixx

Reply:
Yes login_flows is a list of all authorized login flows. I've filled it with an example from the spec but will add Openid-Connect to it when I've understood what it is